pkgs/tools/misc/spigot/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, buildPackages
+, fetchgit
+, autoreconfHook
+, gmp
+, ncurses
+, halibut
+, perl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "spigot";
```

Comment:
I think it should still be `"spigot-${version}"`, or something of that effect.

Reply:
If I do that, then I get 
/nix/store/dy356955k1bvwvvq5878hsfdxgyvjcpz-spigot-20200101-20200101

Which really doesn't seem right.  I'm guessing "name" by default is "${pname}-${version}"?

Reply:
Oh yes you're right! So you have to set `pname` and `version`, then `name` is set automatically. (https://github.com/NixOS/rfcs/pull/35)